pkg/virtctl/pause/pause_test.go

```diff
@@ -27,6 +27,13 @@ var _ = Describe("Pausing", func() {
 		vmiInterface = kubecli.NewMockVirtualMachineInstanceInterface(ctrl)
 	})
 
+	Context("With missing input parameters", func() {
+		It("should fail", func() {
+			cmd := tests.NewRepeatableVirtctlCommand("pause")
```

Comment:
Isn't there a `COMMAND_PAUSE`, like the `COMMAND_EXPOSE` above?

Reply:
fixed